tests/components/todoist/test_calendar.py

```diff
@@ -216,9 +241,6 @@ async def test_calendar_custom_project_unique_id(
     entity = entity_registry.async_get("calendar.all_projects")
     assert entity is None
 
-    state = hass.states.get("calendar.all_projects")
-    assert state.state == "off"
```

Comment:
This assertion was actually incorrect and was only passing due to the invalid logic fixed in this PR.